src/site/content/en/blog/shape-detection/index.md

```diff
@@ -224,6 +201,29 @@ try {
 }
 ```
 
+### Working with the `FaceDetector` {: #facedetector}
+
+The `FaceDetector` always returns the bounding boxes of faces it detects in
+the `ImageBitmapSource`. Depending on the platform, more information
+regarding face landmarks like eyes, nose, or mouth may be available.
```

Comment:
Is this API more about finding where the faces are than about identifying people? Maybe we should say something about that.

Reply:
There's an important difference in literature between face detection (there's *a* face) and face identification (there's a face, and it's *Joe's*). This API is only about detection.